packages/updateDependencies.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+ const fs = require("fs");
+ const path = require("path");
+ const { exit } = require("process");
+ 
+ //=================================================================================================
+ // CLI for programmatically setting the version number and dependencies of all packages before
+ // publishing.  Also used to create links for local development.
+ //
+ // LOCAL DEVELOPMENT:
+ // 
+ //     node updateDependencies.js
+ // 
+ // PUBLISHING:
+ //
+ //     node updateDependencies.js 1.0.0
+ //
+ //=================================================================================================
+ 
+ const localDependencies = {
```

Comment:
We should eventually update this to update the sample version numbers as well. It's ok to skip for now.

Reply:
sure, this initial check in is mostly for PR build, will have to figure out the CI/package publish automation story.